src/Sql/Sql.Test/Sql.Test.csproj

```diff
@@ -11,7 +11,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.25.0-preview" />
+    <Reference Include="Microsoft.Azure.Management.Sql">
```

Comment:
Obviously needs to be updated when new sdk is published

Reply:
Yes, of course.